lib/dl_auth_api_lib/dl_auth_api_lib/views/snowflake.py

```diff
@@ -0,0 +1,43 @@
+from aiohttp import (
+    hdrs,
+    web,
+)
+
+from dl_auth_api_lib.oauth.snowflake import (
+    SnowflakeOAuth,
+    SnowflakeOAuthClient,
+)
+from dl_auth_api_lib.schemas import snowflake as snowflake_schemas
+
+
+class BaseSnowflakeView(web.View):
+    def get_client(self, data: dict[str, str]) -> SnowflakeOAuth:
+        client_settings = SnowflakeOAuthClient(
+            account=data["account"],
+            client_id=data["client_id"],
+            client_secret=data["client_secret"],
+            redirect_uri=data["redirect_uri"],
+        )
+        return SnowflakeOAuth.from_settings(client_settings)
+
+
+class SnowflakeURIView(BaseSnowflakeView):
+    async def post(self) -> web.StreamResponse:
+        req_data = await self.request.json()
+        data = snowflake_schemas.SnowflakeUriRequestSchema().load(req_data)
```

Comment:
why using data instead of query parameters?

Reply:
the same reason as previous